tests/10-EdvClient.spec.js

```diff
@@ -64,6 +64,67 @@ describe('EdvClient', () => {
     docs.length.should.equal(1);
   });
 
+  it('find a document using a multi property query', async () => {
+    const client = await mock.createEdv();
+    client.ensureIndex({attribute: 'content.indexedKey'});
+    client.ensureIndex({attribute: 'content.someKey'});
+
+    const testId = await EdvClient.generateId();
+    const doc = {
+      id: testId,
+      content: {
+        someKey: {
+          b: 5,
+          a: 4
+        },
+        indexedKey: 'value1'
+      }
+    };
+
+    await client.insert({
+      doc,
+      invocationSigner,
+      keyResolver
+    });
+
+    // it should find the document when property keys are in same order.
+    const docs = await client.find({
+      equals: {'content.someKey': {
+        b: 5,
+        a: 4
+      }},
+      invocationSigner
+    });
+
+    docs.should.be.an('array');
+    docs.length.should.equal(1);
+
+    // it should find the document when property keys are in a different order.
+    const docs2 = await client.find({
+      equals: {'content.someKey': {
+        a: 4,
+        b: 5
+      }},
+      invocationSigner
+    });
+
+    docs2.should.be.an('array');
+    docs2.length.should.equal(1);
+
+    // no results when attempting to find a document when
+    // property keys have different values.
+    const docs3 = await client.find({
+      equals: {'content.someKey': {
+        b: 4,
+        a: 5
```

Comment:
If I look closely, I can see that there are, in fact, different values here, but it may be better to make the difference more obvious to a reader/maintainer:

```suggestion
        b: 100,
        a: 200
```

Reply:
How about 11111 and 22222 or something, original and suggestion is still just 1 digit different.